examples/helm/charts/prosody/templates/tests/test-connection.yaml

```diff
@@ -11,5 +11,5 @@ spec:
     - name: wget
       image: busybox
       command: ['wget']
-      args: ['{{ include "prosody.fullname" . }}:{{ .Values.service.port }}']
+      args: ['{{ include "prosody.fullname" . }}:{{ .Values.web.service.port }}']
```

Comment:
@Zempashi I'm not really sure about this change, but without it the template renders invalid url:

```yaml
# Source: jitsi-meet/charts/prosody/templates/tests/test-connection.yaml
apiVersion: v1
kind: Pod
metadata:
  name: "jitsi-meet-prosody-test-connection"
  labels:
    helm.sh/chart: prosody-0.1.0
    app.kubernetes.io/name: prosody
    app.kubernetes.io/instance: jitsi-meet
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/managed-by: Helm
  annotations:
    "helm.sh/hook": test-success
spec:
  containers:
    - name: wget
      image: busybox
      command: ['wget']
      args: ['jitsi-meet-prosody:'] # <--
  restartPolicy: Never
```

```shell
Error: YAML parse error on jitsi-meet/templates/common-configmap.yaml: error converting YAML to JSON: yaml: line 16: could not find expected ':'
helm.go:81: [debug] error converting YAML to JSON: yaml: line 16: could not find expected ':'
YAML parse error on jitsi-meet/templates/common-configmap.yaml
```

Any guidance will be appreciated

Reply:
Indeed I did not try the test. I think it should be `{{ index .Values.service.ports "bosh-insecure" }}`

Reply:
@Zempashi I've updated the PR so it includes your suggestion.

Now the template renders correctly:

```yaml
spec:
  containers:
    - name: wget
      image: busybox
      command: ['wget']
      args: ['jitsi-meet-prosody:5280']
```

Reply:
@Zempashi There's one more location with the same reference to the non-existing port:

https://github.com/Zempashi/docker-jitsi-meet/blob/b73a79fb0dcdb5a536f0ccd267d26f9f4a42c95a/examples/helm/charts/prosody/templates/ingress.yaml#L3

Do you think we should update that one as well ?

Reply:
Why not, but the prosody dependency is not meant to live on it's own today (really tied to jitsi). So this ingress come from the default template the come with helm (`helm create`). So yes this one could be update as well, but this not really mandatory.
If you want to expose the prosody on the ingress you could add route on the jitsi ingress if you want. 
But if you modify this one as well, I will definitely accept it ;)